src/main/java/ui/components/pickers/LabelPickerDialog.java

```diff
@@ -132,6 +145,17 @@ private void initialiseDialog(Stage stage, TurboIssue issue) {
         initModality(Modality.APPLICATION_MODAL); // TODO change to NONE for multiple dialogs
         setTitle("Edit Labels for " + (issue.isPullRequest() ? "PR #" : "Issue #") +
                 issue.getId() + " in " + issue.getRepoId());
+        // MacOS may need this call here to position the dialog properly
+        positionDialog(stage);
```

Comment:
@ndt93 actually i managed to minimize the problem by wrapping the line in a `Platform.runLater`, it helps slightly but i don't think it can be eliminated completely. does it work on windows if we keep this as the only instance of `postitionDialog`? i'm not exactly for tying it to the height property since if the dialog expands having it jump back to the center might be jarring for the user.